src/CTA.WebForms2Blazor/FileConverters/CodeFileConverter.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.WebForms2Blazor.Services;
+
+namespace CTA.WebForms2Blazor.FileConverters
+{
+    public class CodeFileConverter : FileConverter
+    {
+        private readonly WorkspaceManagerService _blazorWorkspaceBuilder;
+        private readonly WorkspaceManagerService _webFormsWorkspaceBuilder;
+
+        public CodeFileConverter(
+            string relativePath,
+            WorkspaceManagerService blazorWorkspaceManager,
+            WorkspaceManagerService webFormsWorkspaceManager
+            ) : base(relativePath)
+        {
+            _blazorWorkspaceBuilder = blazorWorkspaceManager;
+            _webFormsWorkspaceBuilder = webFormsWorkspaceManager;
+
+            _webFormsWorkspaceBuilder.NotifyNewExpectedDocument();
+
+            // TODO: Create class information objects and call
+            // _blazorProjectBuilder.NotifyNewExpectedDocument() in
+            // each one, this is because potentially multiple files
+            // are produced by this file information type
+        }
+
+        protected override byte[] GetFileBytes()
```

Comment:
It won't let it be private since its inherited from an abstract class, the only way to make unaccessible from the other classes is making it protected